### ISSUE TITLE
Bug: Votes breakdown Data not loading

### ISSUE THREAD

**Issue by Da-Colon** [decent-dao/fractal-interface#765]
I had checked on Develop as well. it didn't appear that voting was showing in the UI correctly

![Image](https://user-images.githubusercontent.com/38386583/206888193-a9ac6336-4944-4cc3-9cd7-799eb94f8ccf.png)

**Comment by herbig** [comment]
> I see only 1...

@mudrila yeah we should definitely prevent people without delegated tokens from voting... I'm confused why it shows `< 0.01%` rather than `0%` there though.  That might be another bug
